sirepo/job_driver/__init__.py

```diff
@@ -224,16 +148,11 @@ async def prepare_send(self, op):
         Returns:
             bool: True if the op was actually sent
         """
-        if not self._websocket_ready.is_set():
-            await self._agent_start(op)
-            pkdlog('{} {} await _websocket_ready', self, op)
-            await self._websocket_ready.wait()
-            raise job_supervisor.Awaited()
-        await self.cpu_slot_ready()
-        # must be last, because reserves queue position of op
-        # relative to other ops even it throws Awaited when the
-        # op_slot is assigned.
-        await self.op_ready(op)
+        await self._agent_ready(op)
+        # must be before cpu_slot_q, because reserves queue position
```

Comment:
This comment should reworded to match new architecture and be moved in _slots_ready()

Reply:
374a8cb5